src/BuildConfig.cc

```diff
@@ -97,6 +97,68 @@ BuildConfig::BuildConfig(const std::string& packageName, const bool isDebug)
   }
 }
 
+static std::vector<std::string>
+parseEnvFlags(const std::string& env) {
```

Comment:
Since we just use the return value of `std::getenv` and then iterate over this, I think `std::string_view` is more proper.

Reply:
Actually `std::string_view` can't be used there. 

Although this implementation is really just iterating over the string and seems to be possible just to take slices of it, it is actually because the makefile generator seems not escaping any character including spaces so I decided to reserve the backslashes and quotes temporarily, but they should be removed in fact, something like `\ ` will be replaced with ` ` and `abc"def"ghi` will become `abcdefghi`, etc.

I just thought about this idea again and decided remove the backslashes and quotes after parsing and properly interpret them because reserving them may cause problems when generating compilation database. I don't know if there's an issue about that the makefile generator doesn't escape things correctly but I think there should be an issue or a fix about it (maybe I'll create the issue if it doesn't exist later, if I finished working on the PR). Not going to fix that (I mean the problem that makefile generator doesn't escape properly) in this PR because it's unrelated.